about.html

```diff
@@ -0,0 +1,190 @@
+              </li>
+              <li class="nav-item text-center ps-sm-2 pe-sm-2">
+                <a class="nav-link" href="#">Program</a>
+              </li>
+              <li class="nav-item text-center ps-sm-2 pe-sm-2">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item text-center ps-sm-2 pe-sm-2">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+              <li class="nav-item text-center ps-sm-2 pe-sm-2 d-sm-block d-md-none d-lg-block">
+                <a class="nav-link" href="#">News</a>
+              </li>
+
+              <div class="collapse navbar-collapse justify-content-end ms-2" id="CC-conference-btn-container" >
+                <button type="button" class="text-center border border-4 border-danger p-1 ps-2 pe-2 text-danger ">CC Campaign</button>
+              </div>
+            </ul>
+            
+        </div>
+
+        
+      </div>
+    </nav>
+
+  </header>
+    
+  <main>
+    <section id="headline" class="w-100 position-relative">
+      <img id="img1" width="100%" height="500px" src="./imgs/about.jpg" alt="home image"
+      >
+      <img id="img2" width="100%" height="500px" src="./imgs/about2.jpg" alt="home image"
+      >
+      <div id="home-content" class="position-absolute top-0 h-100 d-flex flex-column justify-content-center align-items-center">
+        <h3 class="text-white d-md-none">
+            EU Largest Tech Conference:
+        </h3>
+
+        <h1 id="conference-title" class="text-white text-center">
+            Visit The Electric & Hybrid <br> 
+            Vehicle Technology Expo Europe:
+        </h1>
+        
+        <div class="line d-none d-lg-block"></div>
+        
+        <p class="text-white text-center d-none d-lg-block">Explore topics in detail with an interactive learning experience on core subject matters in advanced batteries and electric & hybrid vehicles.
+        Explore topics in detail with an interactive learning experience on core subject matters in advanced batteries and electric & hybrid vehicles. <br>
+        Explore topics in detail with an interactive learning experience on core subject matters in advanced batteries and electric & hybrid vehicles.    
+        </p>    
+        <p class="text-white text-center">Meet battery manufacturers, suppliers, engineers, thought leaders and decision-makers for a conference and battery tech expo focused on the latest developments in the advanced battery and automotive industries.</p>
+
+        <h2 id="head-h2" class="text-white text-center">
+            Please contact us per Email for nay further questions<br>
+        </h2>
+            
+        <a href="" class="text-white">#EVTechExpo
+          <img class="p-1 d-none d-lg-inline-block"
+            src="./imgs/icons/ArrowLeftToRight.svg" alt="arrow left to right" />
+        </a>
+            
+        
+        </h2>
+      </div>
+
+    </section>
```

Comment:
- Kindly make sure to stick as much as possible to the given template, which includes:
   - [ ] Increasing the spacing to the left and right of the text so it's more centered.
   - [ ] Wrapping the middle text section to be in a white background container, to match the original template.
   - [ ] Changing the font color of the text to match the given design.
   - [ ] Applying opacity to the background image of this section so that it doesn't oversaturate the section and make the text unreadable.
   
| Your design | Original design |
| --- | --- |
| ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/7bbbb170-1902-40fe-b868-97ad3b123ff3) | ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/1a75588d-5a7c-4484-83a6-7e61a5d79a98) |